src/test/java/org/jitsi/meet/test/StartMutedTest.java

```diff
@@ -57,7 +57,16 @@ public void checkboxesTest()
         driver1.findElement(By.id("startAudioMuted")).click();
         driver1.findElement(By.id("startVideoMuted")).click();
 
-        ensureTwoParticipants();
+        WebParticipant participant = joinSecondParticipant();
```

Comment:
it would be more readable if this var would be named 'participant2' - when reading this I was thinking it's "participant1"